app.rb

```diff
@@ -0,0 +1,159 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def choose_option
+    option = gets.chomp
+    case option
+    when '1'
+      display_books
+    when '2'
+      display_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      display_rentals_by_person_id
+    when '7'
+      puts 'Thank you for using this app!'
+      exit
+    end
+  end
+
+  # rubocop:enable Metrics/CyclomaticComplexity
```

Comment:
Dear reviewers,
I've disabled the linter for this method because I don't know how to fix it. Any help would be appreciated.

![Screenshot 2023-08-31 190905](https://github.com/K0ppai/School_Library/assets/113283174/e58ac728-7eb6-4369-b38f-d8d499fb837c)